dashboards-observability/public/components/application_analytics/helpers/utils.tsx

```diff
@@ -12,5 +12,8 @@ export const isNameValid = (name: string) => {
   if (name.length === 0) {
     toast.push('Name must be at least 1 character');
   }
+  if (name.trim().length === 0) {
+    toast.push('Name must have characters')
```

Comment:
I think you can remove the second `if` since it's also covered here, and message can be `Name must not be empty.` to be more consistent with other plugins?